scripts/clean_slop.py

```
@@ -0,0 +1,71 @@
+                f.write(original_content)
+            print(f"Created backup at {backup_path}")
+
+        with open(out_path, 'w', encoding='utf-8') as f:
+            f.write(content)
+        print(f"Cleaned content saved to {out_path}")
+    else:
+        print("--- Preview of changes ---")
+        # simple diff-like output
+        orig_lines = original_content.splitlines()
+        new_lines = content.splitlines()
+        for i, (orig, new) in enumerate(zip(orig_lines, new_lines)):
```

Ruff (0.15.11)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)